code/game/turfs/simulated.dm

```diff
@@ -52,18 +50,6 @@
 
 /turf/simulated/Entered(atom/A, atom/OL)
 	..()
-	if(ismob(A)) //only mobs make dirt

```

Comment:
Technically, replacing this with `isliving(A)` would have fixed the original issue, but I don't really mind the dirt thing being removed. Where is the dirt even coming from?